GoogleChromeLabs/ps-analysis-tool
# Feature: Create PSAT landing page and dashboard

## Description

Redesign the PSAT landing page and add a dashboard with pinned items and features for easy navigation.

## Relevant Technical Choices

- Remove the iframe and add static content.
- Map internal links for each item in the dashboard.
- Add internal link in PSAT landing page for dashboard.
- Display the landing page when users first install the PSAT extension. On subsequent visits, show the dashboard.
- Add top-level icons in collapsed sidebar.

## Testing Instructions

- Remove and install PSAT extension, you should be able to see the landing page.
- Open extension in a tab, you should now see dashboard.
- All links and buttons in the dashboard should navigate to the related pages.
- Top level icons should be visible in the collapsed sidebar.

## Additional Information:



## Screenshot/Screencast

Landing Page
<img width="1047" alt="image" src="https://github.com/user-attachments/assets/501d3bb9-c55a-4c25-98ba-cb7cc3e78be0">


Dashboard
<img width="1413" alt="image" src="https://github.com/user-attachments/assets/5ab80633-3c27-4833-8420-192645c7c6bb">


---

## Checklist






Fixes #799
